pkg/cmd/scan.go

```diff
@@ -160,6 +162,13 @@ func NewScanCmd() *cobra.Command {
 		false,
 		"Includes cloud provider service-linked roles (disabled by default)",
 	)
+	fl.BoolVar(&opts.Deep,
+		"deep",
+		false,
+		fmt.Sprintf("%s Enable deep mode, disabled by default\n", warn("EXPERIMENTAL:"))+
+			"In deep mode we compare resources internal values to expected ones (like a terraform plan).\n"+
+			warn("This can lead to unexpected behavior (false positives, hidden drifts) when using a tfstate as IaC source and thus this is not recommended"),
```

Comment:
Maybe we can redirect our users to the docs instead of trying to explain what it is in the usage.